### DISCUSSION TITLE
Authorization Header

### DISCUSSION THREAD

**Discussion by davidjnevin** [Jericho/StrongGrid#407] [Q&A] [answered]
Hi Jericho,
Loving what you're doing with Stronggrid, it's working fantastically sending emails.

I'm completely stumped as to how to add an authorization header to the Stronggrid.client to include "Bearer apikey".

Any help would be greatly appreciated.
Thanks so much and sorry for what might be a newbie question.

**Comment by Jericho** [comment, marked as answer]
Thank you for the kind words. I'm glad the library is useful to you.

Regarding the authorization header: the library takes care of it for you, all you have to do is to specify your API key when you instantiate a new instance of the StrongGrid client. Here's an example:

```csharp
var apiKey = "... your api key...";
var strongGridClient = new StrongGrid.Client(apiKey);
```

Hopefully this answers your question.

**Reply by davidjnevin** [reply]
Thanks so much Jericho!
